org.eclipse.xtext.testing/META-INF/MANIFEST.MF

```diff
@@ -22,6 +23,7 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.junit;bundle-version="4.12.0",
  org.eclipse.xtend.lib
 Import-Package: org.apache.log4j;version="1.2.15",
- org.apache.log4j.spi;version="1.2.15"
+ org.apache.log4j.spi;version="1.2.15",
+ org.junit.jupiter.api.extension;version="5.0.0";resolution:=optional
```

Comment:
still 5.0.0?

Reply:
wil you do a pr for umbrella (repackageing of orbit) too

Reply:
I think the lower bound of 5.0.0 is correct here. There are no limitations rgd. usage of JUnit 5.0.0, only that we build against 5.1. The bundle is tested against 5.0, though.

Reply:
am note sure how we test

Reply:
InjectionExtensionTest tests the new extension, or what kind of test do you think of? The lower bound just says that there is no restriction on something that 5.1 added, and works also for 5.0. And the InjectionExtensionTest can be executed also when 5.0.0 is stated in the versions.gradle.